catroid/src/main/java/org/catrobat/catroid/content/XmlHeader.java

```diff
@@ -47,6 +47,7 @@
 
 	private float catrobatLanguageVersion;
 	private boolean landscapeMode;
+	private boolean isCastProject = false;
```

Comment:
This has to be handled when an old project (i.e. one without this field is loaded). have a look at the xml  head and how it is serialized and deserialized.